src/features/import/l10n/messageIds.ts

```diff
@@ -122,6 +122,20 @@ export default makeMessages('feat.import', {
       tags: m('Tags'),
       zetkinHeader: m('Zetkin'),
     },
+    preview: {
+      columnHeader: {
+        ext: m('External ID'),
```

Comment:
I believe this field also exists in the `globalMessageIds`, and otherwise I would call it `ext_id`, `extId` or something like that..